src/main/java/at/hannibal2/skyhanni/features/gui/customscoreboard/ScoreboardEvents.kt

```diff
@@ -158,7 +158,12 @@ enum class ScoreboardEvents(
     ACTIVE_TABLIST_EVENTS(
         ::getActiveEventLine,
         ::getActiveEventShowWhen,
-        "§7(All Active Tablist Events)"
+        "§dHoppity's Hunt\n §fEnds in: §e26h"
```

Comment:
this looks wrong

Reply:
explain

Reply:
is this still the "active tablist events" thing? why is there the hoppity hunt text as template?

Reply:
It is yes, what is your other idea